app/src/main/java/de/danoeh/antennapod/ui/screen/download/DownloadLogDetailsDialog.java

```diff
@@ -1,71 +1,219 @@
         if (!status.isSuccessful()) {
             message = status.getReasonDetailed();
         }
 
-        String humanReadableReason = context.getString(DownloadErrorLabel.from(status.getReason()));
-        SpannableString errorMessage = new SpannableString(context.getString(R.string.download_log_details_message,
+        String humanReadableReason = activity.getString(DownloadErrorLabel.from(status.getReason()));
+        SpannableString errorMessage = new SpannableString(activity.getString(R.string.download_log_details_message,
                 humanReadableReason, message, url));
         errorMessage.setSpan(new ForegroundColorSpan(0x88888888),
                 humanReadableReason.length(), errorMessage.length(), Spannable.SPAN_EXCLUSIVE_EXCLUSIVE);
 
+        CharSequence downloadMessage = getDownloadMessage(podcastTitle, episodeTitle);
+
+        SpannableString clickableDownloadMessage;
+        if (isTitleClickable) {
+            clickableDownloadMessage = getClickableMessage(activity, downloadMessage, feed, onDismissCallback);
+        } else {
+            clickableDownloadMessage = new SpannableString(downloadMessage);
+        }
+
         setTitle(R.string.download_error_details);
-        setMessage(errorMessage);
+        setMessage(TextUtils.concat(clickableDownloadMessage, errorMessage));
         setPositiveButton(android.R.string.ok, null);
-        setNeutralButton(R.string.copy_to_clipboard, (dialog, which) -> {
+        setNeutralButton(R.string.copy_to_clipboard, (copyDialog, which) -> {
             ClipboardManager clipboard = (ClipboardManager) getContext()
                     .getSystemService(Context.CLIPBOARD_SERVICE);
-            ClipData clip = ClipData.newPlainText(context.getString(R.string.download_error_details), errorMessage);
+            ClipData clip = ClipData.newPlainText(activity.getString(R.string.download_error_details), errorMessage);
             clipboard.setPrimaryClip(clip);
             if (Build.VERSION.SDK_INT < 32) {
-                EventBus.getDefault().post(new MessageEvent(context.getString(R.string.copied_to_clipboard)));
+                EventBus.getDefault().post(new MessageEvent(activity.getString(R.string.copied_to_clipboard)));
             }
         });
     }
 
+    @NonNull
+    private SpannableString getClickableMessage(@NonNull Activity activity, CharSequence downloadMessage, Feed feed,
+                                                Runnable onDismissCallback) {
+        SpannableString clickableDownloadMessage = new SpannableString(downloadMessage);
+        ClickableSpan clickableSpan = new ClickableSpan() {
+            @Override
+            public void onClick(@NonNull View widget) {
+                if (feed != null) {
+                    if (feed.getState() == Feed.STATE_SUBSCRIBED) {
+                        Fragment fragment = FeedItemlistFragment.newInstance(feed.getId());
+                        ((MainActivity) activity).loadChildFragment(fragment);
```

Comment:
This can be done with `new MainActivityStarter().withOpenFeed`, then you don't need an activity instance